### ISSUE TITLE
Bugs in Product Engineering Dojo Course

### ISSUE THREAD

**Issue by ponkey1** [dxc-technology/online-pe-dojo#8]
Few of the bugs are as below:
1. After completing the "Getting Started" and "Assessment" modules instead of taking to next module to showing the completion and error message as attached comes. Request to look.

2. Also, while giving assessment, pls mention how the response should be given in the terminal i.e., whether we should write response with parenthesis "( )" or without parenthesis "( )" simply write option like "a" or we should give the complete answer. I faced a challenge that first time I wrote the response with  parenthesis "( )" and after completion is showed that I have scored all answer as incorrect. I attempted again without parenthesis "( )" and was shown a result of 10 out of 10.

3. Also, we are using some old library or methods and on the terminal a message is displayed for the same affecting few lines. 

Rest all looks good, request to look into the same.
![Uploading Err msg.PNG…]()

**Comment by tom-halpin** [comment]
Thank you @ponkey1 will take a look at those issues

**Comment by tom-halpin** [comment]
@ponkey1 I have verified points 1 and 2, I am not seeing any warnings about "old library or methods and on the terminal a message is displayed for the same affecting few lines" can you provide more information as to where you are seeing this specifically the module and page? A screenshot or two would also be helpful

**Comment by ponkey1** [comment]
@tom-halpin , let me elaborate:

The below issues are pertaining to Product Engineering course accessible with the URL:  https://dxc-technology.github.io/about-pe-dojo/modules/

Point 1:
On this Saturday, when I complete these two modules ( "Getting Started" and "Assessment" ), and clicked the "Continue Online Product Engineer Dojo" button to complete these modules on last (Congratulations) screen, in case of both the modules I was getting the error message as shared earlier. 

Today, it is working correctly for "Getting Started" module and taking me to "Assessment" module. But, at the end of the "Assessment" module when I click it takes me back to the first module. Instead of this it should either show me that the course is completed or take better display "Product Engineering - White Belt Dojo". 

This would be correct culmination of the course. Since bug is resolved, so now It is my suggestion.

Point 2:
This is pertaining to "Assessment" module.

I have tried the assessment again with correct answer given with parenthesis "( )" . For example, if the answer a is correct, I wrote (a) instead of a. In case, I use parenthesis "( )" for all answers, I get a message that I have score 0 marks. On the hand,  if I write response without parenthesis "( )", I get a message that I have score 10 out of 10.

My suggestion is to let know audience, how the response should be provided. This will make that participant follows correct model.

Point 3:
This message is more pertaining to the use of depreciated APIs / functions' uses to in the program. A message for the same when we are running conversation is KataCoda UI. This does not affect working, but in future support for these functions may be taken out so better to look.

**Comment by tom-halpin** [comment]
@ponkey1 thanks 

Closing this issue I have added a fix for 1, by redirecting to the welcome, addressed point 2 by adding some instructions to the assessment and am unable to reproduce 3.

Also just as an FYI. This is open sourced content so you and others are welcome to submit a PR with any other improvements you see or fixes for issues you encounter. Indeed PRs from the community were one of the outcomes we were hoping for when open sourcing this content.
